apps/web/src/components/dialog-user-create.tsx

```diff
@@ -0,0 +1,123 @@
+import { Plus } from "lucide-react";
+import { Button } from "./ui/button";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "./ui/dialog";
+
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { z } from "zod";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "./ui/form";
+import { Input } from "./ui/input";
+import { useState } from "react";
+import useUsers from '../hooks/use-users'
+import { useRouter } from "next/router";
+import projectListSwr from '@/hooks/use-project-list'
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "./ui/select";
+
+const formSchema = z.object({
+  email: z.string().email(),
+  projectId: z.string()
+});
+
+type FormData = z.infer<typeof formSchema>;
+
+export function CreateUserDialog() {
+  const [open, setOpen] = useState<boolean>(false);
+  const router = useRouter();
+  const { data, isLoading } = projectListSwr()
+  const { createUser } = useUsers()
+
+  const form = useForm<FormData>({
+    resolver: zodResolver<any>(formSchema),
+    defaultValues: {
+      email: ''
+    },
+  });
+
+  if(!data) return null;
+
+  function onSubmit(values: z.infer<typeof formSchema>) {
+    createUser(values.email, values.projectId)
+  }
+
+  return (
+    <Dialog open={open} onOpenChange={setOpen}>
+      <DialogTrigger asChild>
+        <Button variant="default">
+          <Plus size="20" />
+          Gebruiker aanmaken
+        </Button>
+      </DialogTrigger>
+      <DialogContent>
+        <Form {...form}>
+          <form onSubmit={form.handleSubmit(onSubmit)} className="space-y-4">
+            <DialogHeader>
```

Comment:
Waarom zit de Formfield in de dialog header? 
zou je dit dialog header niet omhoog hoisten zoals
<>
     <DialogHeader>...</DialogHeader/>
     <Form>
         <FormField/>
         <button>
     </Form>
</>

Reply:
Ik had het gebaseerd op jouw widget create, dus ik ging ervan uit dat dit de standaard toepassing is aangezien de form fields in de widget-create dialog ook in de header zaten. Ik zal kijken of ik deze eruit kan tillen.